installing/installing_vsphere/upi/upi-vsphere-preparing-to-install.adoc

```diff
@@ -0,0 +1,35 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="upi-vsphere-preparing-to-install"]
+= Preparing to install a cluster using user-provisioned infrastructure
+include::_attributes/common-attributes.adoc[]
+:context: upi-vsphere-preparing-to-install
+
+toc::[]
+
+You prepare to install an {product-title} on vSphere by:
```

Comment:
Question: should you use an article if you're just saying "OCP" and not "OCP cluster"? If not, I'd either remove the article like this:
```suggestion
You prepare to install {product-title} on vSphere by:
```

Or add "cluster" like this:
```suggestion
You prepare to install an {product-title} cluster on vSphere by:
```